pkgs/applications/video/obs-studio/linuxbrowser.nix

```diff
@@ -0,0 +1,48 @@
+# We don't have a wrapper which can supply obs-studio plugins so you have to
+# somewhat manually install this:
+
+# nix-env -f . -iA obs-linuxbrowser
+# mkdir -p ~/.config/obs-studio/plugins
+# ln -s ~/.nix-profile/share/obs/obs-plugins/obs-linuxbrowser ~/.config/obs-studio/plugins/
```

Comment:
It would be great if we could write an obs-studio-with-plugins derivation. I had a quick go but it doesn't look straight forward to get obs-studio to look into a specific plugins path. I also can't figure out where the plugin paths are specified in the obs-studio code.